database/crudOperations.js

```diff
@@ -0,0 +1,98 @@
+//need to import models into this file
+const {summaries, hosts} = require('./postgresSchema.js');
+
+const insertNewSummaryInfo = async (summaryInfo, cb) => {
+  //should recieve json
+  //summaryBody should include everything except hostName
+  let summaryBody = {...summaryInfo};
+  delete summaryBody.hostName;
+  //hostBody should include only hostName
+  let hostBody = {};
+  hostBody.hostName = summaryInfo.hostName;
+  // //first need to add hostName to hosts model
+  await hosts.create(hostBody)
+   .then((host) => {
+     let hostId =  host.dataValues.id;
+     summaryBody.hostId = hostId;
+    })
+    .catch(err => {
+      console.log(err);
+      //should return 500 for an error or if the record is a duplicate (stayId already in use)
+      cb(500);
+    })
+   await summaries.create(summaryBody)
+   .then((result) => {
+     cb(200)
+   })
+   .catch((err) => {
+     console.log(err);
+     cb(500)
+   })
+}
+
+//read
+const getRoomSummary = async (id, cb) => {
+  let roomId = id;
+
+  const summaryData = await summaries.findOne({
+   where: {"stayId": roomId}
+  })
+
+  if (summaryData === null) {
+    cb(null, 500)
+  } else {
+    let hostId = summaryData.dataValues.hostId;
+    let summaryObj = summaryData.dataValues;
+
+    const hostData = await hosts.findOne({
+      where: {"id": hostId}
+    })
```

Comment:
I would maybe look into Sequelize methods to potentially return data from multiple tables in one query

Reply:
I think you sent us an article for how to do this. However, I struggled a bit trying to implement a query for it. If I have time I will go ahead and try it.